make/harbor.yml.tmpl

```diff
@@ -91,6 +91,7 @@ trivy:
   # https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line
   #
   # github_token: xxx
+  offline_scan: false
```

Comment:
We should describe this new param and how it relates to `skip_update`. You can copy the explanation from https://github.com/aquasecurity/trivy/discussions/1518

Reply:
NIT: Add it as close to `skip_update` property. In air-gapped environment both properties must be set to true so it may help grouping related configuration options.

Reply:
thank you for the comment. the PR is updated now.